migrations/env.py

```diff
@@ -0,0 +1,57 @@
+import configparser
```

Comment:
Ah, I was following `lando-api` which uses a [`migrations`](https://github.com/mozilla-conduit/lando-api/tree/master/migrations) directory. I'll rename it to `alembic`

Reply:
`migrations` in `lando-api` is mostly equivalent to `versions` here -- though it's also a little mangled because we use flask-alembic and not a vanilla Alembic install.